rest_framework_gis/fields.py

```diff
@@ -18,15 +18,29 @@ class GeometryField(Field):
     """
     type_name = 'GeometryField'
 
-    def __init__(self, **kwargs):
+    def __init__(self, precision=None, remove_duplicates=False, **kwargs):
+        self.precision = precision
+        self.rm_dupes = remove_duplicates
```

Comment:
I would really prefer if we could use readable instance attributes, method names and test method names as well, for consistency with the rest of the codebase